src/Frontend/Base.php

```diff
@@ -323,14 +323,14 @@ public function save_default_data( $order_id, $posted_data ) {
 	}
 
 	/**
-	 * Calculate evening fee.
+	 * Calculate delivery day fee.
 	 *
 	 * @param array $order_id ID of order post.
 	 * @param array $posted_data Array of global _POST data.
 	 *
 	 * @return array.
 	 */
-	public function calculate_evening_fee( $order_id, $posted_data ) {
+	public function calculate_delivery_day_fee( $order_id, $posted_data ) {
```

Comment:
`calculate_delivery_day_fee` sounds ambigous to me. At first, I thought this was a function to calculate standard delivery fee. But when I looked into the function, it seems it's a function to calculate morning and evening fee.
How about we rephrase the function name to be like : `calculate_non_standard_fee()`?